src/hooks/useAssetLogoSource.ts

```diff
@@ -61,25 +66,25 @@ export default function useAssetLogoSource(
   isNative?: boolean,
   backupImg?: string | null
 ): [string | undefined, () => void] {
-  const hasWarning = Boolean(address && checkWarning(address))
+  const showLogo = Boolean((address && checkWarning(address, chainId) === null) || isNative)
```

Comment:
should we add a check here to see if the token is one of the common bases besides native?

Reply:
I added that logic to checkWarning so the common bases should always have their image displayed.